flashinfer/mla.py

```
@@ -83,8 +83,10 @@ def _check_trtllm_gen_mla_shape(
     elif kv_cache.ndim != 4:
         raise ValueError(f"Expected kv_cache.ndim == 3 or 4, got {kv_cache.ndim}")
 
-    if qk_nope_head_dim != 128:
-        raise ValueError(f"Expected qk_nope_head_dim == 128, got {qk_nope_head_dim}")
+    if not (qk_nope_head_dim == 128 or qk_nope_head_dim == 192):
+        raise ValueError(
+            f"Expected qk_nope_head_dim == 128 or 192, got {qk_nope_head_dim}"
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)